### ISSUE TITLE
Enbridge Gas: [A11y] Our Company header not accessible via QuickNav

### ISSUE THREAD

**Issue by codyhiggins722** [codyhiggins722/Software-QA-Database.Projects#22]
**Summary:** Dropdown options for the ‘Our Company’ header on enbridgegas.com are not accessible or visible when following VoiceOver instructions for QuickNav.

**Actual Result:** Dropdown options cannot be seen or reached while following the VoiceOver instructions.

**Expected Result:** Dropdown menu options should be fully accessible and navigable while using VoiceOver and QuickNav
